recipes/newrelic/mysql/debian.yml

```diff
@@ -0,0 +1,118 @@
+    prompt: "Please enter your MySQL hostname below. If none is provided, the default value: localhost, will be used."
+    default: "localhost"
+  - name: "NR_CLI_DB_PORT"
+    prompt: "Please enter your MySQL port below. If none is provided, the default value: 3306, will be used."
+    default: 3306
+  - name: "NR_CLI_DATABASE"
+    prompt: "Please enter your MySQL database name below."
+
+install:
+  version: "3"
+
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: create_user_notification
+        - task: setup
+        - task: restart
+        
+    create_user_notification:
+      cmds:
+        - |
+          echo "_ Note: __ You need to have a MySQL user with replication permissions for enabling this integration. If you’d like to create a new user please use these commands as reference. _\n
+          sudo mysql -e \"CREATE USER 'newrelic'@'localhost' IDENTIFIED BY 'YOUR_SELECTED_PASSWORD';\"\n
+          sudo mysql -e \"GRANT REPLICATION CLIENT ON *.* TO 'newrelic'@'localhost' WITH MAX_USER_CONNECTIONS 5;\"\n
+          sudo mysql -e \"GRANT SELECT ON *.* TO 'newrelic'@'localhost' WITH MAX_USER_CONNECTIONS 5;\""
+    setup:
+      label: "Installing mysql integration..."
+      cmds:
+        - |
+          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
+        - |
+          sudo apt-get update
+        - |
+          sudo apt-get install nri-mysql -y
+        - |
+          if [ -f /etc/newrelic-infra/integrations.d/mysql-config.yml ]; then
+            sudo rm /etc/newrelic-infra/integrations.d/mysql-config.yml;
+          fi
+
+          sudo cp /etc/newrelic-infra/integrations.d/mysql-config.yml.sample /etc/newrelic-infra/integrations.d/mysql-config.yml;
+
+        - | 
+          # Check if the provided user can access database.
+          CHECK_DB=$(mysql -u{{.NR_CLI_DB_USERNAME}} -p{{.NR_CLI_DB_PASSWORD}} -e "SHOW GRANTS FOR CURRENT_USER" | grep 'GRANT SELECT' | awk '{print $2, $3}')
+          EXPECTED=$(echo 'SELECT, REPLICATION')
+          if [ "$CHECK_DB" == "$EXPECTED" ] ; then
+            sudo tee /etc/newrelic-infra/integrations.d/mysql-config.yml > /dev/null <<"EOT"
+          integration_name: com.newrelic.mysql
+
+          instances: 
+            - name: mysql-status
+              command: status
+              arguments:
+                hostname: {{.NR_CLI_DB_HOSTNAME}}
+                port: {{.NR_CLI_DB_PORT}}
+                username: {{.NR_CLI_DB_USERNAME}}
+                password: {{.NR_CLI_DB_PASSWORD}}
+                database: {{.NR_CLI_DATABASE}}
+                extended_metrics: true
+                extended_innodb_metrics: true
+                extended_myisam_metrics: true
+                remote_monitoring: true
+          EOT
+          else
+            echo -e "[Error]: Provided user has no access to the desired database.\n - See https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/mysql-monitoring-integration#config for more info."
```

Comment:
@JakubKotkowiak and @devfreddy I believe we also need an `exit 1` in here to signal an error to the newrelic-cli?